### ISSUE TITLE
Accessibility fixes

### ISSUE THREAD

**Issue by gjones** [ClickHouse/click-ui#422]
### Summary
We're trying to ensure that there are no WCAG violations remaining in Click UI.

It looks like a number of components produce the same aria issue: 

Flyout, Splitbutton, Autocomplete, Dropdown, Popover, 
`Elements must only use supported ARIA attributes`
`ARIA attribute is not allowed: aria-expanded="false"`

Also the grid returns this: 
`Scrollable region must have keyboard access`

**Comment by rndD** [comment]
Related: The plugin to check accessibility issues https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd
